lib/connectionContext.ts

```diff
@@ -75,17 +86,54 @@ export namespace ConnectionContext {
    */
   export const userAgent: string = "/js-event-hubs";
 
-  export function create(config: ConnectionConfig, tokenProvider?: TokenProvider): ConnectionContext {
+  function encodeMessage(body: any): any {
+    let result: any;
+    if (body != undefined)
```

Comment:
nit: i typically see this done as `body != null` rather than `body != undefined` (though i think they give you the same result)

Reply:
another stylistic thing: blocks like this without curly braces scare me

Reply:
(if you're interested: https://palantir.github.io/tslint/rules/curly/)